tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -0,0 +1,306 @@
+    user_token = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+    run_scan(cluster, user_token, can_run_scan=False)
+
+
+@if_test_rbac
+def test_rbac_run_scan_project_owner():
+    client, cluster = get_user_client_and_cluster()
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    run_scan(cluster, user_token, can_run_scan=False)
+
+
+@if_test_rbac
+def test_rbac_run_scan_project_member():
+    client, cluster = get_user_client_and_cluster()
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    run_scan(cluster, user_token, can_run_scan=False)
+
+
+@if_test_rbac
+def test_rbac_run_scan_project_read_only():
+    client, cluster = get_user_client_and_cluster()
+    user_token = rbac_get_user_token_by_role(PROJECT_READ_ONLY)
+    run_scan(cluster, user_token, can_run_scan=False)
+
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    aws_nodes = \
+        AmazonWebServices().create_multiple_nodes(
+            5, random_test_name(HOST_NAME))
+    cluster_detail["nodes"] = aws_nodes
+    node_roles = [
+        ["controlplane"], ["etcd"], ["worker"]
+    ]
+    client = get_user_client()
+    cluster = client.create_cluster(name=random_test_name(),
+                                    driver="rancherKubernetesEngine",
+                                    rancherKubernetesEngineConfig=
+                                    rke_config_cis,
+                                    defaultPodSecurityPolicyTemplateId=
+                                    POD_SECURITY_POLICY_TEMPLATE)
+    assert cluster.state == "provisioning"
+    i = 0
+    for i in range(0, 3):
+        aws_node = aws_nodes[i]
+        aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
+        aws_node.execute_command("sudo sysctl -w kernel.panic=10")
+        aws_node.execute_command("sudo sysctl -w kernel.panic_on_oops=1")
+        if node_roles[i] == ["etcd"]:
+            aws_node.execute_command("sudo useradd etcd")
+        docker_run_cmd = \
+            get_custom_host_registration_cmd(client, cluster, node_roles[i],
+                                             aws_node)
+        aws_node.execute_command(docker_run_cmd)
+    time.sleep(5)
+    cluster = validate_cluster_state(client, cluster)
+
+    # the worklods under System project to get active
+    time.sleep(10)
+    cluster_detail["cluster"] = cluster
+
+
+def verify_cis_scan_report_skipped(report_link, skip_test, token):
+    head = {'Authorization': 'Bearer ' + token}
+    response = requests.get(report_link, verify=False, headers=head)
+    report = response.json()
+    report_results = report["results"]
+    # print(report_results)
+    report_results[0]
+
+
+def verify_cis_scan_report(report_link, token):
+    head = {'Authorization': 'Bearer ' + token}
+    response = requests.get(report_link, verify=False, headers=head)
+    report = response.json()
+    print(report)
+    assert report["total"] == 97, "Incorrect number of tests run"
```

Comment:
Do we expect this number of tests to change in the future? If so, move it to a variable at the top of the file so it can be clearly seen.

Reply:
Yes it might change. Moved it